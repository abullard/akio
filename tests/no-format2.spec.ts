import { describe, it, expect } from "vitest";
import { spawnWrapper } from "./test-utils";

describe('no-format2.spec.ts', () => {
    it('should turn formatting off, --no-format', async () => {
        const stubbedCommandForNoOp = 'test:stub';
        const skipInput = '-i';
        const cmd = "pnpm";
        const args = ['akio', '--no-format', skipInput, stubbedCommandForNoOp];

        const { stdout } = await spawnWrapper(cmd, args);

        expect(stdout).toMatchSnapshot();
    });
});
