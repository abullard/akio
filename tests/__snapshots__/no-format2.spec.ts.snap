// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`no-format2.spec.ts > should turn formatting off, --no-format 1`] = `
"
> @abullard/akio@1.0.0 akio /root/Developer/git/abullard/akio
> npx akio --no-format -i test\\:stub

pnpm akio
	-----
Found scripts matching: "test:stub"
1. test:stub  â€” DO NOT USE - stubbed npm script for snapshot tests"
`;
